auditwheel/wheel_abi.py

```diff
@@ -116,7 +117,65 @@ def analyze_wheel_abi(wheel_fn: str):
     log.debug('external reference info')
     log.debug(json.dumps(external_refs, indent=4))
 
+    # get versioned symbols from external_refs
+    # {realpath: libname} e.g.
+    # {'/path/to/external_ref.so.1.2.3': 'external_ref.so.1'}
+    external_ref_no_linux = {}
+    # go through all policies
+    for e in external_refs.values():
```

Comment:
I would much prefer proper variable names in the loops instead of single letter names. I'm surprised the linter doesn't complain about this.